app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,14 @@
+<% provide(:title, 'Home') %>
+<!DOCTYPE html>
+<html>
```

Comment:
Please review our code in TWITTER clone or Parent-Child relationship lessons regarding Ruby on Rails .html.erb files